src/coreclr/src/tools/Common/TypeSystem/Common/Utilities/GCPointerMap.cs

```diff
@@ -0,0 +1,272 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Text;
+
+using Debug = System.Diagnostics.Debug;
+
+namespace Internal.TypeSystem
+{
+    /// <summary>
+    /// Represents a bitmap of GC pointers within a memory region divided into
+    /// pointer-sized cells.
+    /// </summary>
+    public partial struct GCPointerMap : IEquatable<GCPointerMap>, IComparable<GCPointerMap>
+    {
+        // Each bit in this array represents a pointer-sized cell.
+        private int[] _gcFlags;
```

Comment:
Nit - uint might better convey the fact that this is bitmask.

Reply:
I ported this from CoreRT. I can see if we can quickly switch over to uints instead, sure.

Reply:
Please don't spend too much time on it, if it turns out to be viral for whatever reason, there's just no reason to waste time on it. Thanks for taking a look nonetheless.

Reply:
It wasn't that viral - I adjusted them to be uints.